features/visitor_can_navigate_the_site.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Visitor can navigate the site
+    As a visitor,
+    In order to navigate the site
+    I would like to use a navbar
+
+    Scenario:
+        Given I visit the landing page
+        Then I should see "ABOUT US"
+        And I should see "MENU"
+        And I should see "Opening hours"
+        And I should see "Park location"
+        Then I click "ABOUT US" link
```

Comment:
This should be "When"

Reply:
OK. Changed to 'When'. Edited the cucumber steps also.